pkg/metrics/metrics.go

```diff
@@ -500,6 +508,10 @@ func ReportEvictedWorkloads(cqName kueue.ClusterQueueReference, reason string) {
 	EvictedWorkloadsTotal.WithLabelValues(string(cqName), reason).Inc()
 }
 
+func ReportAggregatedWorkloadSlices(cqName kueue.ClusterQueueReference) {
```

Comment:
```suggestion
func ReportReplacedWorkloadSlices(cqName kueue.ClusterQueueReference) {
```
nit, just to align naming